graylog2-web-interface/src/components/streamrules/StreamRuleForm.jsx

```diff
@@ -1,18 +1,21 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { Col } from 'components/graylog';
-import { Input } from 'components/bootstrap';
-import BootstrapModalForm from 'components/bootstrap/BootstrapModalForm';
+import { Col, Well } from 'components/graylog';
+import { Input, BootstrapModalForm } from 'components/bootstrap';
 import { TypeAheadFieldInput, Icon } from 'components/common';
 import { DocumentationLink } from 'components/support';
+import HumanReadableStreamRule from 'components/streamrules//HumanReadableStreamRule';
```

Comment:
Could you adjust the `//`?

Reply:
I will adjust this, after merging the PR. I don't want to block the merge, because of this little change.